canonical/netplan.io
# Convert reference content back to Markdown

## Done
Convert the reference page source back to Markdown
Added a Markdown template to render the pages with a client-side rendered table of contents

## QA
- Open the demo
- Click on References in the navigation
- See that page renders nicely
- See the Table of content load and render correctly
- Check all other pages work as expected
